gemrb/core/Interface.cpp

```diff
@@ -250,6 +254,11 @@ Interface::Interface()
 	SpecialSpellsCount = -1;
 	SpecialSpells = NULL;
 	Encoding = "default";
+	#ifndef WIN32
+	SystemEncoding = nl_langinfo(CODESET);
+	#else
+	SystemEncoding = "UTF-16";
```

Comment:
You can't do this in C++11 anymore. I suggest to use `std::string` that and feed its `.c_str()` where required.

Reply:
You meant `SystemEncoding` not being `const`? Right, how about adding `const` modifier, instead of using `std::string`?

Reply:
Sure, if it suits.